app/src/main/java/org/oppia/app/topic/conceptcard/ConceptCardFragment.kt

```diff
@@ -29,7 +29,8 @@ class ConceptCardFragment : InjectableDialogFragment() {
     }
   }
 
-  @Inject lateinit var conceptCardFragmentPresenter: ConceptCardFragmentPresenter
+  @Inject
```

Comment:
Please revert this change to
 `@Inject lateinit var conceptCardFragmentPresenter: ConceptCardFragmentPresenter`